NEWS.md

```diff
@@ -11,6 +11,8 @@
 * correctly handle selectors of the form `:col => AsTable` and `:col => cols`
   by expanding a single column into multiple columns
   ([#2780](https://github.com/JuliaData/DataFrames.jl/pull/2780))
+* if `subset!`
```

Comment:
Seems like a few words are missing. :-D

Reply:
I hoped it was clear what the PR does 😆.

BTW: it turns out that this change was needed. The check I added to `GroupedDataFrame` consistency already caught one bug in our testsets :).